_api-files/embedding-api/documentation/introduction.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Introduction
+content-type: "embed-doc"
+order: 1
+
+sections:
+  - content: |
+      The {{ page.api-name }} can be used to:
+
+        - Create Stitch client accounts,
+        - Access existing Stitch client accounts, and
+        - Configure source and destination connections within those clients' accounts
+
+      The {{ page.api-name }} is organized around REST and uses resource-oriented URLs to promote usability. Each endpoint uses standard HTTP verbs like `GET` and `POST`, and we will return [standard HTTP response codes](#response-codes) to indicate request status or errors.
+
+      We built the {{ page.api-name }} to accept and return [JSON](http://json.org) in all responses, including errors.
+
+
+  - title: "Obtain Partner Credentials"
+    anchor: "obtain-partner-credentials"
+    content: "To use the API, you need to register your application and obtain partner credentials. Reach out to [{{ page.contact-email }}](mailto:{{ page.contact-email }}) to get started."
```

Comment:
We might actually want to refer them to the [future] form on the marketing site at this point. See [the very end of Lee's content doc here](https://docs.google.com/document/d/1r3ny7NfLB6oWCh_um5O1ZSv23woLMcPa9K45CO9SYTY/edit).